api/envoy/config/trace/v2/trace.proto

```diff
@@ -198,6 +200,19 @@ message OpenCensusConfig {
   repeated TraceContext outgoing_trace_context = 9;
 }
 
+// Configuration for AWS X-Ray tracer.
+message XRayConfig {
+  // The endpoint of the X-Ray Daemon where the spans will be sent. Since by default daemon
+  // listens to localhost:2000, so the default value is 127.0.0.1:2000.
+  string daemon_endpoint = 1 [(validate.rules).string.min_bytes = 1];
+
+  // The custom name to name a X-Ray segment. By default will use cluster name.
```

Comment:
It looks like we try and pull the app mesh virtual node name by default. That isn't documented here. This will always be the cluster name anyway. So we should probably just stop checking for that environment variable. No reason to tie this to appmesh specifically.

Reply:
>So we should probably just stop checking for that environment variable. No reason to tie this to appmesh specifically.

sounds good to me.

>This will always be the cluster name anyway.

I need more clarification please.

Reply:
When we generate the initial config for the envoy (bootstrap config) we set the `node.cluster` to `/mesh/<MeshName>/virtualNode/<virtualNodeName>`. An AppMesh virtual node and an Envoy cluster are basically 1:1 mapped.

Reply:
And `APPMESH_VIRTUAL_NODE_NAME` is the source of this value. https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html